src/App.js

```diff
@@ -1,11 +1,23 @@
-import React from "react";
+import React from 'react';
 import Header from "./components/Header.js";
-
-
+import { Link } from 'react-router-dom'
+import CharacterList from "./components/CharacterList"
+import WelcomePage from './components/WelcomePage'
+import { Route } from 'react-router-dom'
 export default function App() {
+
   return (
     <main>
       <Header />
+      <Link to='/welcome'>
+        WelcomePage
+      </Link>
+      <Link to='/characterList'>
+        CharacterPage
+      </Link>
+    <Route exact path='/' component={CharacterList}/>
+    <Route path='/characterList' component={CharacterList}/>
+    <Route path='/welcome' component={WelcomePage}/>
```

Comment:
Your welcome page should be going to the home "/" path and the character list should be 1 route, like this:
```
<Route exact path='/' component={WelcomePage}/>
<Route path='/characterList' component={CharacterList}/>
```